front/src/components/Messages.js

```diff
@@ -55,14 +55,21 @@ const MessagesDiv = styled.div`
 
 
 const Messages = (props) => {
+  const [isNavigationMode, setNavigationMode] = useState()
   const messagesEndRef = useRef(null)
 
   useEffect(() => {
-    messagesEndRef.current.scrollIntoView({ behavior: "smooth" })
+    !isNavigationMode && messagesEndRef.current.scrollIntoView({ behavior: "smooth" })
   }, [props.messages])
 
+  const handleScroll = (scrollEvent) => {
+      var element = scrollEvent.currentTarget
+      var isBottom = element.scrollTop + element.offsetHeight === element.scrollHeight
+      isBottom ? setNavigationMode(false) : setNavigationMode(true)
```

Comment:
This is logically equivalent to
`setNavigationMode(isBottom)`

Reply:
oh you're right. let me fix this.

Reply:
Sorry, it should of course be `setNavigationMode(!isBottom)`.

Reply:
Done. Changed `isBottom` to `!isBottom` though.